components/IndexPage/iconMapper.js

```diff
@@ -0,0 +1,27 @@
+import stringSimilarity from "string-similarity";
+import { Icons } from "theme/icons";
+
+export default (property) => {
+  let bestMatch;
+  switch (property) {
+    case "amount":
+    case "amountOfThisGood":
+      return "folder";
+    case "price":
+      return "euro-sign";
+    case "storageOption":
+      return "check-double";
+    case "creatorPublicKey":
+      return "key";
+    case "transactionId":
+      return "info-circle";
+    default:
+      bestMatch = stringSimilarity.findBestMatch(property, Object.keys(Icons))
+        .bestMatch.target;
+
+      if (bestMatch.startsWith("fab")) bestMatch = bestMatch.slice(3);
+      if (bestMatch.startsWith("fa")) bestMatch = bestMatch.slice(2);
+
+      return bestMatch.charAt(0).toLowerCase() + bestMatch.slice(1);
+  }
```

Comment:
Here we try to add icons for the standard properties and will just select closest icon name for other properties.

Reply:
@aaaaref Sounds interesting! You can also get some inspiration from Farshid's design on the icons:
![Frame 1215](https://user-images.githubusercontent.com/71267015/96930264-49da7380-14c8-11eb-96ad-9733de1be980.jpg)